tests/integration/awslambda/test_lambda_api.snapshot.json

```diff
@@ -8021,7 +8021,7 @@
       "publish_layer_version_exc_invalid_runtime_arch": {
         "Error": {
           "Code": "ValidationException",
-          "Message": "2 validation errors detected: Value '[invalidruntime]' at 'compatibleRuntimes' failed to satisfy constraint: Member must satisfy enum value set: [nodejs12.x, java17, provided, nodejs16.x, nodejs14.x, ruby2.7, python3.10, java11, dotnet6, go1.x, nodejs18.x, provided.al2, java8, java8.al2, dotnetcore3.1, python3.7, python3.8, python3.9]; Value '[invalidarch]' at 'compatibleArchitectures' failed to satisfy constraint: Member must satisfy constraint: [Member must satisfy enum value set: [x86_64, arm64]]"
```

Comment:
Based on this output, we should actually be able to remove the `nodejs12.x`/`dotnetcore3.1` runtimes as well at some point..

Reply:
Yes, they are not available anymore for new function creation. We already removed them from the multi runtime tests, as they are not verifiable against AWS anymore, so they are not tested anymore, but left the support in for now, in case people want to test their legacy applications for some reason.

We currently do not have any structure for removing the support, as far as I know.